release_notes.md

```diff
@@ -4,19 +4,7 @@
 - My change description (#PR)
 -->
 - Increase timeout for http proxying requests [#9433](https://github.com/Azure/azure-functions-host/pull/9433)
-
 - Update Python Worker Version to [4.14.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.14.0)
-- Update Java Worker Version to [2.12.1](https://github.com/Azure/azure-functions-java-worker/releases/tag/2.12.1)
-- Update PowerShell Worker 7.0 to 4.0.2850 [Release Note](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v4.0.2850)
-- Update Node.js Worker Version to [3.8.0](https://github.com/Azure/azure-functions-nodejs-worker/releases/tag/v3.8.0)
-- Identity dependencies updated to 6.31.0:
-  - Microsoft.IdentityModel.Tokens
-  - System.IdentityModel.Tokens.Jwt
-  - Microsoft.IdentityModel.Abstractions
-  - Microsoft.IdentityModel.JsonWebTokens
-  - Microsoft.IdentityModel.Logging
-- Updated Grpc.AspNetCore package to 2.55.0 (https://github.com/Azure/azure-functions-host/pull/9373)
-- Update protobuf file to v1.10.0 (https://github.com/Azure/azure-functions-host/pull/9405)
 - Send an empty RpcHttp payload if proxying the http request to the worker (https://github.com/Azure/azure-functions-host/pull/9415)
```

Comment:
I see lines 20-22 in the previous version's release notes.
https://github.com/Azure/azure-functions-host/blob/release/4.x/release_notes.md
Is there any reason they are still included here?

Reply:
Updated the release notes